lib/mcts/node.rb

```diff
@@ -1,13 +1,13 @@
 module MCTS
   class Node
-    attr_reader :parent,  :move, :wins, :visits, :children, :game_state
+    attr_reader :parent, :move, :wins, :visits, :children, :game_state
 
     def initialize(game_state, move, parent)
       @parent        = parent
       @game_state    = game_state
       @move          = move
       @wins          = 0.0
-      @visits        = 0
+      @visits        = Float::EPSILON
```

Comment:
? What's this change doing here? Doesn't seem to be correct and it's also what broke the tests :)

Reply:
@PragTob ohh yeah that protected from a divide by zero... outside of scope from the PR though, i'll remove.